### ISSUE TITLE
Unable to add CIFS/SMB root folder: Resource temporarily unavailabl

### ISSUE THREAD

**Issue by bushwald** [Sonarr/Sonarr#6723]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Current Behavior

Hello and thank you contributors for this wonderful software.  I'm having some trouble after reinstalling my system.

I'm setting up Sonarr again after reinstalling my OS and I'm no longer able to select the root folder mounted from a Synology NAS via CIFS.  The folder is mounted.  I can navigate the folder and see its files and their permissions from the shell (both inside and outside the container).  But when I select it as a root folder I see the following error:

<img width="1260" alt="image" src="https://github.com/Sonarr/Sonarr/assets/7196340/7e28fee5-494d-45fc-b463-4f39b6e65538">

### Expected Behavior

The root folder should be added and scanned

### Environment

```markdown
- OS: Ubuntu Server 20.04
- Sonarr: 4.0.4.1491
- Docker Install: yes, lscr.io/linuxserver/sonarr:latest
- Using Reverse Proxy: no
- Browser: Firefox 124.0.2 
- Database:
```

### What branch are you running?

Main

### Trace Logs?

``` shell
[Error] RootFolderService: Unable to get free space and unmapped folders for root folder /movies/
radarr      |
[v5.4.6.8723] System.InvalidOperationException: Resource temporarily unavailabl
 ---> Mono.Unix.UnixIOException: Resource temporarily unavailabl [EWOULDBLOCK].
   --- End of inner exception stack trace ---
   at Mono.Unix.UnixDriveInfo.Refresh(Boolean throwException)
   at Mono.Unix.UnixDriveInfo.Refresh()
   at Mono.Unix.UnixDriveInfo.get_AvailableFreeSpace()
   at NzbDrone.Mono.Disk.ProcMount.get_AvailableFreeSpace() in ./Radarr.Mono/Disk/ProcMount.cs:line 23
   at NzbDrone.Mono.Disk.DiskProvider.GetAvailableSpace(String path) in ./Radarr.Mono/Disk/DiskProvider.cs:line 53
   at NzbDrone.Core.RootFolders.RootFolderService.<>c__DisplayClass15_0.<GetDetails>b__0() in ./Radarr.Core/RootFolders/RootFolderService.cs:line 225
   at System.Threading.ExecutionContext.RunFromThreadPoolDispatchLoop(Thread threadPoolThread, ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
   ```

**Comment by thezoggy** [comment]
1) you reported bug with radarr, this is sonarr.
2) this isnt an issue with \*arr but your os

cifs bug in kernel, downgrade or wait until update end of the month
> Confirmed bug with CIFS+Kernel 5.15.0-102 - https://bugs.launchpad.net/ubuntu/+source/linux/+bug/2060780 Fix will be in 5.15.0-104 by the end of the month.

**Comment by bushwald** [comment]
Ahh so it is.  I was having the same issue with both and got mixed up.  Thanks for your help pointing me to the kernel bug.
